### ISSUE TITLE
Disable animations via adb

### ISSUE THREAD

**Issue by dpgraham** [appium/appium-espresso-driver#72]
Espresso doesn't work if animations are not disabled.

Add command to disable them via shell

```
# Turn off animations
adb shell settings put global window_animation_scale 0 &
adb shell settings put global transition_animation_scale 0 &
adb shell settings put global animator_duration_scale 0 &
```

**Comment by KazuCocoa** [comment]
You can also disable animation with https://github.com/appium/io.appium.settings as the following.

```
$ adb shell pm grant io.appium.settings android.permission.SET_ANIMATION_SCALE
$ adb shell am broadcast -a io.appium.settings.animation --es setstatus disable
```

**Comment by dpgraham** [comment]
Fixed by #73 by @mykola-mokhnach
